### ISSUE TITLE
Upgrade to up-to-date Twitter Bootstrap (4.0)

### ISSUE THREAD

**Issue by yaskevich** [jonorthwash/ud-annotatrix#367]
That allows to use adavanced UI controls. Also fixes issues of previous versions.

**Comment by jonorthwash** [comment]
Why "Twitter"?

**Comment by yaskevich** [comment]
Yeah, right. Actually, 'cause I remember the times. Now it is Bootstrap by Bootstrap team.
Fixed by #356. So it became possible to use new handy control ("switch"), now it's on home page.
![2019-06-26 21_23_07-UD Annotatrix - Chromium](https://user-images.githubusercontent.com/522902/60204914-97a18600-9858-11e9-957a-fed3973caa19.png)

**Comment by jonorthwash** [comment]
Ah, that could be useful!
